crt_portal/cts_forms/views.py

```diff
@@ -182,6 +183,38 @@ def _related_reports_count(report):
         return 1
 
 
+def _get_email(report):
+    return report.contact_email
```

Comment:
I'm not sure this function adds much for readability. Maybe just directly access contact_email, instead?

If this was for the `map` function, then the comprehension should help in this particular case. In other cases a lambda would also work:

e.g., if for some reason a comprehension wasn't available:

```py
map(lambda report: report.contact_email, reports)
```